rooms/wc.cpp

```diff
@@ -56,38 +57,19 @@ Room wc(__attribute__((unused)) GameData& gameData) {
 
   if (yes()) {
```

Comment:
It doesn't have any sense. You demand player to say yes/no to a statement. Maybe change text into question?